config/transform-elasticsearch/documents/datadiscovery_study.json

```diff
@@ -41,14 +41,57 @@
         }
       ]
     },
+    "entryType": {
+      "{list}": [
+        {
+          "{or}": [
+            {
+              "{replace}":{
+                "possible_terms": [
+                  "Genotyping", "Genotyping Study",
+                  "allele size", "genotype"
+                ]
+              },
+              "{with}": {
+                "replaced_by": "Genotyping Study"
+              }
+            },
+            {
+              "{replace}":{
+                "possible_terms": [
+                  "Phenotypes", "Phenotyping", "Field Experiement",
+                  "Greenhouse (29\u00baC/20\u00baC)", "Green house",
+                  "Growth chamber", "Phenotyping Study", "Provenance trial",
+                  "Half sibling progeny trial", "Clonal trial", "Progeny trial",
+                  "Other", "Provenance and half sibling progeny trial",
+                  "Species comparison", "Seed orchard", "Demonstration test",
+                  "Full sibling progeny trial", "Juveniles comparison",
+                  "Clonal archiva, clone bank", "Conservation plot",
+                  "Biomasse test - sylvabiom", "Response study", "raw"
```

Comment:
I think there is an extra "e" at `biomass`.

Reply:
Yep, but once again we leave as is